internal/provider/resource_database.go

```diff
@@ -22,7 +22,7 @@ var availableCloudProviders = []string{
 	"azure",
 }
 
-var databaseCreateTimeout = time.Minute * 20
+var databaseCreateTimeout = time.Minute * 40
 var databaseReadTimeout = time.Minute * 5
 var databaseDeleteTimeout = time.Minute * 20
 var databaseUpdateTimeout = time.Minute * 20
```

Comment:
Maybe change the Update timeout as well. In particular, when you add regions, Updates take a while.

Reply:
Ok, I changed the update timeout to be 40 also.